packages/mobile/package.json

```diff
@@ -7,16 +7,16 @@
     "author": "IOTA Foundation <contact@iota.org>",
     "license": "Apache-2.0",
     "scripts": {
-        "build": "NODE_ENV=production PLATFORM=mobile webpack",
+        "build": "NODE_ENV=production PLATFORM=mobile webpack && npm assets:alpha",
         "ios": "cap copy ios && cap open ios",
         "ios:update": "cap sync ios && cap open ios",
         "android": "cap copy android && cap open android",
         "android:update": "cap sync android && cap open android",
         "dev": "NODE_ENV=development PLATFORM=mobile webpack serve --content-base public --host 0.0.0.0 --client-log-level none",
-        "dev:device": "NODE_ENV=development PLATFORM=mobile webpack --stats && cap copy",
-        "icons:alpha": "icons/icongen.sh icons/alpha.png",
-        "icons:beta": "icons/icongen.sh icons/beta.png",
-        "icons:prod": "icons/icongen.sh icons/prod.png"
+        "dev:device": "NODE_ENV=development PLATFORM=mobile webpack --stats &&&& cap copy",
```

Comment:
Should this just be `&&` instead of `&&&&`?

Reply:
yes thanks! will fix 😅